components/monitoring/logging/staging/configure-logforwarder-patch.yaml

```diff
@@ -0,0 +1,20 @@
+---
+- op: add
+  path: /spec/outputs/-
+  value:
+    name: splunk-receiver
+    secret:
+      name: splunk-forwarder-secret
+    type: splunk 
+    url: https://http-inputs-rhcorporate.splunkcloud.com
+- op: add
+  path: /spec/pipelines/-
+  value:
+    name: fluentd-splunk-logforward
+    inputRefs:
+      - application
+      - infrastructure
+      - audit
```

Comment:
We can try this but the Splunk team might not want all of these.  Can you ask them if this is OK after it's enabled?

Reply:
Sure. We can always remove any type of logs if we decide they are not useful for us.